README.md

```diff
@@ -16,6 +16,9 @@ A wrapper to make it really easy to deal with iOS Keychain and store your user's
 - Store credentials under an **Access Group to enable Keychain Sharing**.
 - **TouchID/FaceID integration** with a reusable `LAContext` instance. 
 
+## Important Notices
```

Comment:
The requirements section does not match this requirement.

Reply:
That is, the updated requirements supersede the notice. If we'd prefer to use the notice, then we should keep the old requirements.